lib/quicly.c

```diff
@@ -1118,6 +1162,9 @@ static quicly_conn_t *create_connection(quicly_context_t *ctx, const char *serve
     quicly_loss_init(&conn->egress.loss, conn->super.ctx->loss,
                      conn->super.ctx->loss->default_initial_rtt /* FIXME remember initial_rtt in session ticket */);
     conn->egress.path_challenge.tail_ref = &conn->egress.path_challenge.head;
+    conn->egress.send_ack_at = INT64_MAX;
+    update_cc_ticks(conn->super.ctx);
+    cc_init(&conn->egress.cc.ccv, &newreno_cc_algo, 1280 * 8, 1280);
```

Comment:
Is newreno_cc_algo a function?

Reply:
It's a list of callback functions, defined at https://github.com/h2o/daemons-cc/blob/master/cc_newreno.c#L58-L64.